packages/gatsby-theme-aio/src/components/GetCredential/Organization.js

```diff
@@ -126,6 +126,13 @@ const Organization = () => {
                         & > div > div > ul > li > div > div {
                           height: 20px !important;
                         }
+
+                        & > div > div > ul > li > span > div > p {

```

Comment:
Why are we using this way of selecting the elements? Won't it be better to use a class or id?

Reply:
It generates the common class. So If we change that using the class name it will affect the overall picker so I'm using like 
this

Reply:
Is there a way around this? I'm worried if we move elements around, this will break.